src/style.css

```diff
@@ -0,0 +1,136 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-direction: row;
+  font-family: inherit;
+  font-style: normal;
+  margin: 28px;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container {
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+}
+
+.card__imagis {
+  padding-bottom: 36px;
+  padding-left: 2.5px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0;
+  width: 166px;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  padding-top: 4px;
+  padding-bottom: 16px;
+  margin: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars-reviews {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__stars {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__stars:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.card__reviews {
+  padding-left: 17px;
```

Comment:
Maybe would be better to push apart reviews and start with flex and `justify-content: space-between`?